docs/platform/howto/saml/setup-saml-jumpcloud.rst

```diff
@@ -0,0 +1,71 @@
+Set up SAML with JumpCloud
+===========================
+
+This article explains how to set up SAML with `JumpCloud <https://jumpcloud.com/>`_ for an organization in Aiven. For more information on SAML and instructions for other identity providers, see the :doc:`Set up SAML authentication </docs/platform/howto/saml/saml-authentication>` article.
+
+Prerequisite steps in Aiven Console
+------------------------------------
+
+#. In the organization, click **Admin**.
+
+#. Select **Authentication**.
+
+#. Click **Add authentication method**.
+
+#. Enter a name and select SAML. You can also select the teams that users will be added to when they sign up or log in through this authentication method.
+
+You are shown two parameters needed to set up the SAML authentication in JumpCloud:
+
+* Metadata URL
+* ACS URL
+
+Configure SAML on JumpCloud
+----------------------------
+
+#. In the `JumpCloud admin console <https://console.jumpcloud.com/login>`_, go to **SSO**.
+
+#. Select **Custom SAML App**.
+
```

Comment:
Set ``IdP Entity ID``. This is an unique case-sensitive identifier.

Reply:
How is this different from step 4 in this section: "Set the SAML Entity ID to the IdP Entity ID from JumpCloud."?

Reply:
Apologies for missing this. @staceysalamon-aiven The first step is "Set IdP Entity ID" is on Jumpcloud's console, while step 4 is on Aiven's console. 

Step 4 is about copying the URL from step 1 onto Aiven "SAML Entity ID" field

Reply:
@chong-aiven I'm certain I understood this correctly; I added a step to set the IdP entity ID after going to the customer SAML app in JumpCloud.